cmd/start.go

```diff
@@ -83,14 +86,22 @@ var startCmd = &cobra.Command{
 		} else {
 			level = zapcore.InfoLevel
 		}
-		l, err := logger.NewZapLogger(level)
+		l, err := logger.NewZapLogger(level, viper.GetString(prettyLogFlagName))
 		if err != nil {
 			log.Fatalf("can't initialize zap logger: %v", err)
 		}
 		logger := logger.NewLogger(l, Debug)
 		rtLogger := logger.WithFields(zap.String("component", "start"))
 
 		if viper.GetString(syncProviderFlagName) != "" {
+			rtLogger.Warn("DEPRECATED: the --sync-provider flag has been deprecated " +
```

Comment:
It looks like this was duplicated during a rebase. Could you please doublecheck this section?

Reply:
done
btw, could you tell me why it could occur every time I committed?